code/common/pubsub.q

```diff
@@ -104,17 +104,22 @@ closesub:{[h]
   delhandlef[;h]each t;
  };
 
+/ Scope value to inside the function so we can iterate over a set of tables without loading
+/ them all into memory at once
+/ Takes a symbol to a global table
+extractschema: {t: value x; $[.Q.qp t; t; 0#t]};
```

Comment:
```suggestion
extractschema:{t:value x; $[.Q.qp t; t; 0#t]};
```

mostly a personal preference stylistic thing, but we generally don't have spaces after `:` anywhere in torq

Reply:
Got it, I'll make the changes, thanks